### PR TITLE
add: linkedin service

### PR DESCRIPTION
**Pull request by ld0rman**
This commit adds the linked in service to likely. Complete with popup, count and icon.

### CODE REVIEW

**Comment by Proghat** [comment]
@ld0rman, thank you for contribution! Just tested it and sharing works good. Going to test counts.
screen shot 2016-01-28 at 12 45 09 am

I see few issues:
1. Don't know who should build new release. @ld0rman, you could do it by `gulp build`.
2. Can you please add linkedin color for the icon? See how it implemented for other services: https://github.com/ilyabirman/Likely/blob/master/styles/likely.styl#L157.

File: source/services/linkedin.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+module.exports = {
+    counterUrl: 'http://www.linkedin.com/countserv/count/share?url={url}&format=jsonp&callback=?',
+    convertNumber: function (counter) {
+        return counter.count;
+    },
+    popupUrl: 'https://www.linkedin.com/shareArticle?url={url}&mini=true&title={title}',
+    popupWidth: 600,
+    popupHeight: 500
+};
```
Comment (Proghat) [inline source/services/linkedin.js]: I'm not sure about style guide of this repository but I personally prefer to add newline at the end of each file.

Comment (ld0rman) [inline source/services/linkedin.js]: Happy to add eol, though it should be noted that this is not the case across the repo.

**Comment by Proghat** [comment]
@ld0rman, I've tested counts but request from my site to linkedin was blocked for security reasons. Browser blocks requests from https to http. I think we can just change schema in `couterUrl` https://github.com/ilyabirman/Likely/pull/16/files#diff-33addb891f804ae4d3d1964c8d2b1683R6 and it will work fine.

File: source/services/linkedin.js (hunk 1 above)
Comment (Proghat) [inline source/services/linkedin.js]: Thank you :). Right. I've created issue to discuss it with maintainers: https://github.com/ilyabirman/Likely/issues/17.

**Comment by ld0rman** [comment]
Made the protocol dynamic like in the other services. Was this what you meant?

**Comment by Proghat** [comment]
Thank you! It works.
screen shot 2016-02-01 at 11 00 36 am

**Comment by ezhlobo** [comment]
@volter9 ping for review

**Comment by untone** [comment]
@Proghat Why dont you using a relative sheme?

**Comment by ezhlobo** [comment]
@untone probably because the pull-request has not been merged yet.

**Comment by iamakulov** [comment]
Hi! I’m a new maintainer of Likely.

@ld0rman, @Proghat, thank you a lot for your work! I’m sorry, but we aren’t going to add LinkedIn to Likely. Adding the button is easy with your PR, but it will increase the script size and make us responsible for supporting and updating the button in case of API changes. We think the LinkedIn button will be used really rarely, so it isn’t a good deal.

Again, thank you for your work! If you think you need a new button, feel free to submit an issue so we can discuss the idea there before implementing it.

**Pull request closed without merging**